packages/roosterjs-content-model/lib/publicTypes/event/PasteElementProcessor.ts

```diff
@@ -0,0 +1,16 @@
+import { ContentModelBlockGroup } from '../group/ContentModelBlockGroup';
+import { DomToModelContext } from '../context/DomToModelContext';
+
+/**
+ * A function type to process HTML element when do DOM to Content Model conversion in Paste Event.
+ * If the function returns true, means that the element processing was handled by the function.
+ * @param group Parent content model group
+ * @param element The element to process
+ * @param context The context object to provide related information
+ */
+
+export type PasteElementProcessor<T extends Node> = (
```

Comment:
What is the difference between this interface and ElementProcessor?

Reply:
Only that the return type needs to be a boolean.

Reply:
In that case, I prefer change the common processor type to force it return a boolean and change all exiting processors to return true. Although this will change more files, but it makes things consitent.

Reply:
I think instead of that I will just pass in a domToModelOption to all the event so we can change the behavior as needed.